<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceSettingsPage</class>
 <widget class="QWidget" name="AppearanceSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Appearance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="use_system_color_set">
        <property name="text">
         <string>Use the system default color set</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="use_a_custom_color_set">
        <property name="text">
         <string>Use a custom color set</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="select_foreground_color_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select foreground color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_foreground_color">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="select_background_color_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select background color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_background_color">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>use_a_custom_color_set</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_background_color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_a_custom_color_set</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_background_color_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_a_custom_color_set</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_foreground_color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_a_custom_color_set</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_foreground_color_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
